### PR TITLE
adding datadog and lograge

### PR DESCRIPTION
**Pull request by AlyHKafoury**

### CODE REVIEW

File: docker/datadog-agent/conf.d/nginx.d/conf.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    # - type : (mandatory) type of log input source (tcp / udp / file)
+    #   port / path : (mandatory) Set port if type is tcp or udp. Set path if type is file
+    #   service : (mandatory) name of the service owning the log
+    #   source : (mandatory) attribute that defines which integration is sending the logs
+    #   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
+    #   tags: (optional) add tags to each logs collected
+
+  - type: file
+    service: ararat
```
[the remark this answers is not in the document]
Comment (AlyHKafoury) [reply docker/datadog-agent/conf.d/nginx.d/conf.yml]: you are correct

File: docker/datadog-agent/conf.d/nginx.d/conf.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    # - type : (mandatory) type of log input source (tcp / udp / file)
+    #   port / path : (mandatory) Set port if type is tcp or udp. Set path if type is file
+    #   service : (mandatory) name of the service owning the log
+    #   source : (mandatory) attribute that defines which integration is sending the logs
+    #   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
+    #   tags: (optional) add tags to each logs collected
+
+  - type: file
+    service: campaign-forms
+    path: /var/log/nginx/datadog.log
```
Comment (mattdrees) [inline docker/datadog-agent/conf.d/nginx.d/conf.yml]: sorry for jumping in here, but this looks off. Are the nginx_access logs really called datadog.log?

Comment (twinge) [reply docker/datadog-agent/conf.d/nginx.d/conf.yml]: Yes, I asked the same question on a similar PR for ararat. That's apparently what I decided to call that log file when I did the first implementation that these others are copying from.

Comment (mattdrees) [reply docker/datadog-agent/conf.d/nginx.d/conf.yml]: do you want to leave this as-is, then, or go back and change your mind?

Comment (twinge) [reply docker/datadog-agent/conf.d/nginx.d/conf.yml]: My recollection is that I did that so we could have parallel logging where `datadog.log` had the new logging format, and the old format of access logs were still being generated for whatever log management was previously looking for. The prior implementation was sending directly to syslog and not writing files, so there wouldn't be a conflict in writing an 'access.log' file.

I have a light preference for going back to calling it `access.log` now that we're implementing broadly, but feel bad for creating more work.

Comment (mattdrees) [reply docker/datadog-agent/conf.d/nginx.d/conf.yml]: @jbirdjavi do you care one way or the other?

Comment (jbirdjavi) [reply docker/datadog-agent/conf.d/nginx.d/conf.yml]: I don't care, but if we're going to change it, it has to be now, cause all of the apps are starting to get updated with this logging solution.

Comment (mattdrees) [reply docker/datadog-agent/conf.d/nginx.d/conf.yml]: I'd encourage us to change it now, then. It doesn't seem like that much work to me.

**Comment by jbirdjavi** [comment]
@AlyHKafoury Looks like travis is failing again.  I think the global_registry gem is one that does need to be upgraded.

**Pull request merged**